azerothcore/azerothcore-wotlk
# feature(World/Config): Implement MaxWhoListReturns

## CHANGES PROPOSED:
-  Implement MaxWhoListReturns config


## ISSUES ADDRESSED:
- Closes https://github.com/azerothcore/azerothcore-wotlk/issues/2026



## TESTS PERFORMED:
- Builds
- tested in-game
Before:
![image](https://user-images.githubusercontent.com/24550914/84591078-d189e480-ae3b-11ea-9c86-c54121f36ef4.png)
After:
![image](https://user-images.githubusercontent.com/24550914/84591045-a30c0980-ae3b-11ea-8909-241158e24cc9.png)




## HOW TO TEST THE CHANGES:

1. Use 2 accounts
2. Check /who (Remember, don't use GM ON)
3. Change config to 1
4. See that only 1 out of 2 shows



## Target branch(es):








 
## How to test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here in the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR
